beetmoverscript/test/test_script.py

```diff
@@ -100,17 +103,27 @@ def test_move_beets(event_loop):
     context.task = get_fake_valid_task()
     context.properties = get_fake_balrog_props()["properties"]
     context.properties['platform'] = context.properties['stage_platform']
-    manifest = generate_candidates_manifest(context)
+    context.artifacts_to_beetmove = get_upstream_artifacts(context)
+    manifest = generate_beetmover_manifest(context.config, context.task, context.properties)
 
     expected_sources = [
-        'https://queue.taskcluster.net/v1/task/VALID_TASK_ID/artifacts/public/build/target.package',
-        'https://queue.taskcluster.net/v1/task/VALID_TASK_ID/artifacts/public/build/en-US/target.package'
+        os.path.abspath(
```

Comment:
Need to actually create/touch dummy files for {target.mozinfo.json, target.txt, target_info.txt} under test/test_work_dir/cot/eSzfNqMZT_mSiQQXu8hyqg/public/build in order for tests to pass